Solutions/RubrikSecurityCloud/Playbooks/RubrikAnomalyAnalysis/azuredeploy.json

```diff
@@ -22,8 +23,8 @@
             "6. Repeat steps for other connections"
         ],
         "prerequisitesDeployTemplateFile": "Azure-Sentinel/Solutions/RubrikSecurityCloud/Playbooks/RubrikCustomConnector/Rubrik_custom_conn.json",
-        "lastUpdateTime":  "2022-09-26T00:00:00.000Z",
-        "entities":  ["account","url"],
+        "lastUpdateTime":  "2022-01-20T00:00:00.000Z",
+        "entities":  ["account","url","api"],
```

Comment:
Hi @benmeadowcroft "API" isn't a valid entity, please check the following document --> https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#entity-mapping